links/src/utils/getLinks.ts

```diff
@@ -31,7 +31,10 @@ type GetLinks = {
 export const getLinks: GetLinks = async () => {
 	try {
 		const slug = `cdn/links`;
-		const { data } = await storyblok.get(slug);
+		const { data } = await storyblok.get(slug, {
+			token: "api token",
+		});
+
```

Comment:
```suggestion
		const { data } = await storyblok.get(slug);

```
Following best practices the API token should be added using environment variables, commonly in a .env file.

An example of this is in `links/src/lib/storyblok.ts`

Reply:
[Why We Shouldn't Commit Secrets](https://littlemaninmyhead.wordpress.com/2021/04/05/why-we-shouldnt-commit-secrets-into-source-code-repositories/)